streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorContextImplTest.java

```diff
@@ -384,12 +387,14 @@ public void localSessionStoreShouldNotAllowInitOrClose() {
     }
 
     @Test
-    public void shouldNotSendRecordHeadersToChangelogTopic() {
+    public void shouldSendV0RecordHeadersToChangelogTopic() {
```

Comment:
We'll want to restore this test back to exactly the old version of this diff, to be sure we don't have a regression on older brokers.

Reply:
And we probably want to add a new test that we write the consistency header if the flag is enabled, right?